### PR TITLE
[mongo] Updated the declaration of service check tags to include those input …

### PR DESCRIPTION
**Pull request by avaughan89**
…into the yaml config file

Tested locally and confirmed in app here:

http://cl.ly/300R3G3M3p14

### CODE REVIEW

**Comment by MrTravisB** [comment]
Any chance you do this for the memcache check as well?

https://github.com/DataDog/dd-agent/blob/master/checks.d/mcache.py#L325

    service_check_tags = ["host:%s" % server, "port:%s" % port] + custom_tags

**Comment by avaughan89** [comment]
@MrTravisB For that check it would look like:

`service_check_tags.extend(custom_tags)` 

with similar placement.  I believe we will look into adding this into all our checks.

**Comment by olivielpeau** [comment]
Just a note: this service check already has a `host` tag that gets moved to a `server` tag in the backend, so with this change each service check will end up with 2 `server` tags (they can be seen on the screenshot you provided actually).

Not sure if it's something that could make the service check harder to use, but it's definitely something we need to take into account before merging.

**Comment by olivielpeau** [comment]
Just amended the commit so that service checks don't have a duplicate `server` tag.

**Comment by avaughan89** [comment]
@olivielpeau Good catch.  Thanks for updating.

File: checks.d/mongo.py [hunk 1]
```diff
@@ -576,29 +576,28 @@ def total_seconds(td):
         db_name = parsed.get('database')
         clean_server_name = server.replace(password, "*" * 5) if password is not None else server
 
-        additional_metrics = instance.get('additional_metrics', [])
+        if not db_name:
+            self.log.info('No MongoDB database found in URI. Defaulting to admin.')
```
Comment (remh) [inline checks.d/mongo.py]: Let's log in debug. Otherwise this will spam the log for little information.

Comment (remh) [inline checks.d/mongo.py]: Just realized it was in info in the past. We should still log this to debug.

**Comment by truthbk** [comment]
@olivielpeau any consensus on the usability of the additional `server` tag? Personally I don't think it makes all that sense given there's already a matching server tag available for the host.

**Comment by olivielpeau** [comment]
just had a quick chat with @truthbk: I did amend the commit so that there wouldn't be a duplicate `server` tag on the service check, so this is good to go!

Merging

**Pull request merged**